ROMFS/px4fmu_common/init.d/rc.vehicle_setup

```diff
@@ -38,11 +39,14 @@ if [ $VEHICLE_TYPE == mc ]
 then
 	if [ $MIXER == none ]
 	then
-		echo "Mixer undefined"
+		echo "Multicopter mixer undefined"
```

Comment:
Detail: shorter messages will reduce flash usage.

Reply:
> One more thing to ask if you'd like to see done, I can remove lines 50-62 and 141-145. Because the default value is now being set first, the special cases that result in setting the same value can be deprecated.

If you checked all the airframes that they set MAV_TYPE correctly, then yes we can remove this logic.

Reply:
The one that's definitive and simple is
```
if [ $MIXER == caipirinha_vtol ]
then
	set MAV_TYPE 19
fi
```
The `MAV_TYPE` is already expressly set in `130001_caipirhana_vtol`, and any aircraft config file outside of the PX4/Firmware repo depending on that block of code will still result in the same `MAV_TYPE` being set.  Because not every multicopter config file that sets the `MIXER` variable  sets the `MAV_TYPE` variable, there is more risk with corner cases in multicopter airframes.  I can take care of the VTOL code mod in this PR and will submit the multicopter work as a follow-on PR!

Updates as discussed and rebase on current master pushed up. Thanks @bkueng !